spec/models/iteration_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'rails_helper'
+require_relative '../../app/models/iteration'
```

Comment:
Why this is was necessary?

Reply:
Because without this require for now, all tests are be getting from Central::Support::Iteration class